sky/backends/onprem_utils.py

```diff
@@ -178,23 +178,53 @@ def check_and_install_local_env(ips: List[str], auth_config: Dict[str, str]):
         ips, *ssh_credentials)
     sky_ray_version = constants.SKY_REMOTE_RAY_VERSION
 
+    def _version_compare(cur_version: str, min_version: str) -> bool:
+        """Returns true if current version exceeds minimum version."""
+        cur_ver_lst = cur_version.split('.')
+        min_ver_lst = min_version.split('.')
+        cur_ver_lst = [int(i) for i in cur_ver_lst]
+        min_ver_lst = [int(i) for i in min_ver_lst]
+        n = len(cur_ver_lst)
+        m = len(min_ver_lst)
+        if n > m:
+            for i in range(m, n):
+                min_ver_lst.append(0)
+        elif m > n:
+            for i in range(n, m):
+                cur_ver_lst.append(0)
+        for i in range(len(cur_ver_lst)):
+            if cur_ver_lst[i] > min_ver_lst[i]:
+                return True
+        return False
+
```

Comment:
Should we use `packaging.version` directly, as we already used it in https://github.com/skypilot-org/skypilot/blob/e9b378e63296e0a545bc83e6f54e05e4a0afd765/sky/backends/wheel_utils.py#L22?
https://packaging.pypa.io/en/stable/version.html

Reply:
Wow, this is a great suggestion! Thanks @Michaelvll, made it a lot shorter :)